tests/chainer_tests/distributions_tests/test_bernoulli.py

```diff
@@ -156,5 +163,16 @@ def test_double_backward_gpu(self):
             cuda.to_gpu(self.logit), cuda.to_gpu(self.x),
             cuda.to_gpu(self.gy), cuda.to_gpu(self.ggx))
 
+    def test_backward_where_logit_has_infinite_values(self):
+        self.logit += numpy.inf
+        with numpy.errstate(invalid='ignore'):
+            log_prob = distributions.bernoulli._bernoulli_log_prob(
+                self.logit, self.x)
+
+        log_prob.backward()
+
+        # just confirm that the backward method runs without raising error.
+        assert True
```

Comment:
How does the line confirm it?

Reply:
I'm sorry for misleading code.

Is the following without assert statement clearer?
```
# just confirm that the backward method runs without raising error.
log_prob.backward()
```

What I wanted to express is following.
> This test method just confirms that the backward method runs without raising error.
> So assert statement is always passed.
> `assert True` explicitly expresses there is nothing to assert.

Reply:
In the repository `assert True` is not used.  How about replacing the last 4 lines with the following?
```
        # just confirm that the backward method runs without raising error.
        log_prob.backward()
```

Reply:
I adopted that. Thank you.